src/tests/ftest/util/soak_test_base.py

```diff
@@ -451,12 +455,18 @@ def job_completion(self, job_id_list):
                     self.log.info(
                         "<< Job %s failed with status %s>>", job, result)
             # gather all the logfiles for this pass and cleanup test nodes
-            command = "/usr/bin/rsync -avtr --min-size=1B {} {}/".format(
-                self.sharedsoaktest_dir, self.outputsoak_dir)
-            command2 = f"/usr/bin/rm -rf {self.sharedsoaktest_dir}"
+            cmd = f"/usr/bin/rsync -avtr --min-size=1B {self.soak_log_dir} {self.outputsoak_dir}/"
+            cmd2 = f"/usr/bin/rm -rf {self.soak_log_dir}"
+            if self.enable_remote_logging:
+                result = run_remote(self.log, self.hostlist_clients, cmd, timeout=600)
+                if result.passed:
+                    result = run_remote(self.log, self.hostlist_clients, cmd2, timeout=600)
+                if not result.passed:
+                    self.log.info("Remote copy failed with %s", result.log_output)
```

Comment:
`result.log_output(Logger)` is a method that requires an argument and does not return an output - it logs messages.  Since this output will already be in the job.log at this point I'd recommend:
```suggestion
                    self.log.info("Remote copy failed on %s", result.failed_hosts)
```

Reply:
Also, maybe `self.log.error`? @mjean308 Do you manually inspect logs for these errors?